pkg/printers/internalversion/printers_test.go

```diff
@@ -1655,12 +1663,12 @@ func TestPrintPod(t *testing.T) {
 					},
 				},
 			},
-			[]metav1alpha1.TableRow{{Cells: []interface{}{"test5", "1/2", "podReason", 6, "<unknown>"}}},
+			[]metav1alpha1.TableRow{{Cells: []interface{}{"test5", "1/2", "podReason", 6, "<unknown>", "<none>", "<none>"}}},
 		},
 	}
 
 	for i, test := range tests {
-		rows, err := printPod(&test.pod, printers.PrintOptions{ShowAll: true})
+		rows, err := printPod(&test.pod, printers.PrintOptions{ShowAll: true, Wide: true})
```

Comment:
We should additional testcases instead of modifying existing testcase which is already testing without wide!

Reply:
Yes，sir. I,m used to think the test with wide will well cover the test without wide.Thank you for your advice，I will make the changes accordingly.